### PR TITLE
Added: violationID in details UI

### LINKED ISSUE #884: Display ViolationID in the Details UI
Display the ViolationID in the Policy Violations Detail UI

### PR DESCRIPTION
**Pull request by arifa11111**
# Description

Display ViolationID in the Details UI

Fixes #884 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Chore (no code changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] My commit message/PR follows the contribution guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

List any documentation updates that are needed for the Wiki

#### Before Modification
Screenshot from 2023-03-15 12-59-22

#### After Modification
Screenshot from 2023-03-15 15-06-03

### CODE REVIEW

File: webapp/src/app/pacman-features/secondary-components/policy-violation-desc/policy-violation-desc.component.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ export class PolicyViolationDescComponent implements OnInit {
   selectedDomain: string;
   accordionData: any;
   labelData: any;
+  ViolationIDParams: any;
```
Comment (abdulkudduszemoso) [inline webapp/src/app/pacman-features/secondary-components/policy-violation-desc/policy-violation-desc.component.ts]: Can we give proper data type here if possible, other then "any"

File: webapp/src/app/pacman-features/secondary-components/policy-violation-desc/policy-violation-desc.component.ts [hunk 2]
```diff
@@ -38,6 +38,7 @@ export class PolicyViolationDescComponent implements OnInit {
   selectedDomain: string;
   accordionData: any;
   labelData: any;
+  Params: any;
```
Comment (astra1) [inline webapp/src/app/pacman-features/secondary-components/policy-violation-desc/policy-violation-desc.component.ts:41]: class properties should be written in camelCase

File: webapp/src/app/pacman-features/secondary-components/policy-violation-desc/policy-violation-desc.component.html [hunk 3]
```diff
@@ -21,7 +21,13 @@
             {{violationData.policyViolated}}
         </div>
     </div>
-    <div class="policy-violation-label flex flex-col" *ngIf="violationData.policyDescription">
+    <div class="policy-violation-label flex flex-col" *ngIf="Params.issueId">
+        <div class="policy-violation-label-header capitalize">
+            violation ID</div>
+        <div class="policy-violation-label-content" [innerHTML]="Params.issueId" title="{{Params.issueId}}">
```
Comment (astra1) [inline webapp/src/app/pacman-features/secondary-components/policy-violation-desc/policy-violation-desc.component.html:27]: if `Params.issueId` doesn't contain any [entities](https://developer.mozilla.org/en-US/docs/Glossary/Entity), than you shouldn't use [innerHTML] property binding, use instead one-way binding, eg. `<div class="policy-violation-label-content" ...>{{ params.issueId }}</div>`

**Review by astra1 — CHANGES_REQUESTED** [review]

File: webapp/src/app/pacman-features/secondary-components/policy-violation-desc/policy-violation-desc.component.html [hunk 4]
```diff
@@ -21,7 +21,13 @@
             {{violationData.policyViolated}}
         </div>
     </div>
-    <div class="policy-violation-label flex flex-col" *ngIf="violationData.policyDescription">
+    <div class="policy-violation-label flex flex-col" *ngIf="violationId">
+        <div class="policy-violation-label-header capitalize">
+            violation ID</div>
```
Comment (astra1) [inline webapp/src/app/pacman-features/secondary-components/policy-violation-desc/policy-violation-desc.component.html:26]: please, capitalize the first letter `violation ID` to `Violation ID`

**Review by plyubich — APPROVED** [review]
good solution - approved

**Review by sidharthjain-zemoso — APPROVED** [review]

**Review by santhosh-challa — APPROVED** [review]

**Comment by PaladinCloudJohn** [comment]
Thank you, @arifa11111, for being a contributor. Congrats on your first Paladin Cloud merge.

**Pull request merged**